packages/components/Badge/src/Badge.tsx

```diff
@@ -23,8 +23,7 @@ export const badgeLookup = (layer: string, userProps: BadgeProps): boolean => {
     layer === userProps['shape'] ||
     (!userProps['shape'] && layer === 'circular') ||
     layer === userProps['badgeColor'] ||
-    (I18nManager.isRTL && layer === 'rtl') ||
-    (layer === 'shadowToken' && (!userProps.appearance || userProps.appearance === 'filled' || userProps.appearance === 'tint'))
+    (I18nManager.isRTL && layer === 'rtl')
```

Comment:
So it seemed like this code was supposed to be checking the variant to see if a shadow was added or not, but this didn't seem to work. I didn't see any difference removing it so i just removed it, but if anyone has any insight what this code should be doing/how to fix it let me know